src/style.css

```diff
@@ -1 +1,100 @@
-/* add styles here */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: 'Avenir', sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+  height: 768px;
+}
+
+.icon {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px;
+  padding-left: 62px;
+  display: block;
+}
+
+.icon-2 {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 12px;
+  padding-left: 33px;
+  background-size: 1.1%;
+  display: block;
+}
```

Comment:
create common class for icon as well
and fix icon for small input, it seems too big